samples/MyEf.Hr/docs/9-Service-Bus-Publish.md

```diff
@@ -170,8 +170,12 @@ To achieve a basic test within a developers machine then the API host should be
 
 <br/>
 
-## Conclusion
+## Verify
 
 At this stage we now have our events being published to Azure Service Bus. This essentially concludes the `Hr` domain functionality, with respect to enabling the requisite APIs and publishing of corresponding events (where applicable).
 
-Next we will create a new _Security_ domain that will perform a [Service Bus Subscribe](./Service-Bus-Subscribe.md) of the _Termination_ related events and proxy [Okta](https://www.okta.com/) (as the fictitious company's identity solution) automatically _Deactivating_ the Employee's account.
+*Verification Steps TBD*
```

Comment:
What is the TBD?

Reply:
Removed for now.  I haven't looked at any other validation/verification steps yet on pages 8-10.